wisp/wisp-rate-limiting/src/main/kotlin/wisp/ratelimiting/RateLimiter.kt

```diff
@@ -27,13 +27,37 @@ interface RateLimiter {
     amount: Long = 1
   ): ConsumptionData
```

Comment:
Exceptions for flow control on this are bad news

Reply:
Ahh, I’m misreading the docs. ‘This will raise’ is not raising if the consumption fails, but raising if the underlying datastore is unreachable. We don’t typically document exceptions like that.

Reply:
Noted, I can see how that would be confusing. Will remove this the next time I touch this code